README.md

```diff
@@ -1,6 +1,53 @@
 # BMX
 
-BMX grants you API access to your AWS accounts, based on Okta credentials that you already own.
-It uses your Okta identity to create short-term AWS STS tokens, as an alternative to long-term IAM access keys.
+BMX provides API access to your AWS accounts using existing Okta credentials. Instead of long-term IAM access keys, BMX creates short-term AWS STS tokens using your Okta identity.
 
-This is a work in progress dotnet 7 branch
+## Installation
+
+Download the appropriate binary from the [releases](https://github.com/Brightspace/bmx/releases) page. For D2Lers, visit [bmx.d2l.dev](https://bmx.d2l.dev) for installation.
+
+## Commands
+
+### configure
+
+Set up the global BMX configuration file with the following command:
+```PowerShell
+bmx configure --org your-okta-organization --user your-okta-username
+```
+### print
+
+To setup AWS credentials as environment variables in PowerShell, run:
+```PowerShell
+bmx print --account account-name --role role-name | iex
+```
+Or in Bash/sh/Zsh, run:
+```Bash
+$(bmx print --account account-name --role role-name)
+```
+
+### write
+
+Create a new AWS credentials profile with the following command:
+```Powershell
+bmx write --account account-name --role role-name --profile my-profile
+```
+You can use your created profile by configuring any supporting AWS client. For example, for the AWS CLI :
+```Powershell
+aws sts get-caller-identity --profile my-profile
+```
+### notes
```

Comment:
Right now the "notes" heading looks like another command... This may create more distinction visually:

```suggestion

#### Notes
```